src/github.com/couchbase/sync_gateway/db/sequence_id.go

```diff
@@ -223,7 +236,7 @@ func (s *SequenceID) UnmarshalJSON(data []byte) error {
 			if err != nil {
 				return err
 			}
-			if strings.Contains(raw, "-") {
+			if strings.Contains(raw, "-") || strings.Contains(raw, ".") {
```

Comment:
could you add a comment on the meaning of `raw` containing `-` or `.` and ideally give examples or point to docs that explain the format of the Sequence ID?